examples/openai-fga/scripts/fga-init.ts

```diff
@@ -26,40 +26,12 @@ async function main() {
     },
   });
 
-  // 01. WRITE MODEL
```

Comment:
Why are we removing the model creation?

Reply:
because the idea is that the README.md (#5 ) and blog post would teach how do it instead.